integration_tests/test_suites/dagster-azure-live-tests/integration_tests/test_compute_log_manager.py

```diff
@@ -28,18 +36,38 @@ def test_compute_log_manager(
         ["dagster", "asset", "materialize", "--select", "my_asset", "-m", "azure_test_proj.defs"],
         check=True,
     )
+    logs_captured_record = DagsterInstance.get().get_event_records(
+        EventRecordsFilter(
+            event_type=DagsterEventType.LOGS_CAPTURED,
+        )
+    )[0]
+
+    stdout, stderr = (
+        get_captured_logs_from_urls(
+            logs_captured_record,
+            credentials,
+        )
+        if dagster_yaml.name not in YAMLS_NOT_CAPTURED
+        else get_captured_logs_from_run(
+            logs_captured_record,
+            prefix_env,
+            container_client,
+        )
+    )
+
+    assert stdout.count("Printing without context") == 10
+    assert stderr.count("Logging using context") == 10
+
+
+def get_captured_logs_from_urls(
+    captured_logs_event: EventLogRecord, credentials: ClientSecretCredential
+) -> tuple[str, str]:
     logs_captured_data = check.not_none(
-        DagsterInstance.get()
-        .get_event_records(
-            EventRecordsFilter(
-                event_type=DagsterEventType.LOGS_CAPTURED,
-            )
-        )[0]
-        .event_log_entry.dagster_event
+        captured_logs_event.event_log_entry.dagster_event
     ).logs_captured_data
 
-    assert logs_captured_data.external_stderr_url
-    assert logs_captured_data.external_stdout_url
+    assert logs_captured_data.external_stderr_url is not None
```

Comment:
nit: "" should succeed?

Reply:
I'm explicitly just dealing with the type checker here, which complains about Nonetypes being passed to the next few lines.

"" will subsequently fail when we actually query azure